ianalyzer_readers/readers/rdf.py

```diff
@@ -0,0 +1,45 @@
+from typing import Iterable
```

Comment:
This module is not included in the documentation; it should be added in `/docs/api.md`. I would also recommend adding a docstring at the top of the module.